sway-error/src/diagnostic.rs

```diff
@@ -70,8 +70,24 @@ impl Diagnostic {
         self.labels_in_source(self.issue.source_path().unwrap())
     }
 
+    const HELP_NONE: &'static str = "__$$_^^__HELP__NONE__^^_$$__";
```

Comment:
I understand the need for something, but this is hacky. Couldn't you make this work with `Option` or empty strings if you can't rework the logic to simply do nothing?